src/classes/classes-private.ts

```diff
@@ -0,0 +1,58 @@
+//private classes
+
+class EmployeePrivate {
+  constructor(private identifier: string) {}
+}
+
+class FinanceEmployeePrivate extends Employee {
+  getFinanceIdentifier() {
+    return `fin-${this.identifier}`;
+  }
+}
+
+class Reptiles extends Animal {
+  numberOfLegs: number;
+
+  constructor(animalType: string, color: string, numberOfLegs: number) {
+    super(animalType, color);
+    this.numberOfLegs = numberOfLegs;
```

Comment:
can this be improve with the new syntax of ommiting from definition & implementation?

Refer comment in protected file